### PR TITLE
Add a glossary entry for JSON.

### PR DESCRIPTION
**Pull request by Julian**
Refs: json-schema-org/community#199

### CODE REVIEW

File: learn/glossary.md [hunk 1]
```diff
@@ -36,6 +36,18 @@ While future drafts may introduce new behavior or changes to existing behavior,
 
 The current list of drafts can be found [here](https://json-schema.org/specification-links.html#published-drafts).
 
+### JSON
+
+A pervasive data interchange format used for representing and transmitting data as human readable text.
+JSON is extremely widely used, and parsers which can read and write it exist for essentially every commonly-used programming language.
+
+JSON Schema, distinctly, is built *on top* of JSON, in that JSON [schemas](#schema) are themselves JSON objects which describe other JSON objects.
+The two are, however, entirely different pieces of the conceptual puzzle, with JSON being a concrete format for *representing* data and JSON Schema being a way to *schematize* data which is written in the JSON format.
```
Comment (jdesrosiers) [inline learn/glossary.md:45]: I think something we want to convey in a glossary entry for JSON is that JSON Schema doesn't require that your schemas or instances are JSON, just that they are compatible with JSON. I think the current wording implies a tighter connection to JSON than is currently accurate.

Comment (Julian) [reply learn/glossary.md:45]: That's mentioned in the entry for schemas [below on the page](https://github.com/json-schema-org/json-schema-org.github.io/pull/521/files#diff-1034555c346947deb021c12fdf836bf2610faf33f61c03dbabd97345f3820181R94) -- think we should repeat it here as well?

Comment (jdesrosiers) [reply learn/glossary.md:45]: I don't think we need to give a full explanation like below, but I think adding some qualifiers would help. I think people don't generally read a glossary top to bottom, they jump around to the terms they don't know. So, we can't assume they read other parts.

Example: "which is written in the JSON format" => "which is written in a JSON-compatible format".

File: learn/glossary.md [hunk 2]
```diff
@@ -36,6 +36,18 @@ While future drafts may introduce new behavior or changes to existing behavior,
 
 The current list of drafts can be found [here](https://json-schema.org/specification-links.html#published-drafts).
 
+### JSON
+
+A pervasive data interchange format used for representing and transmitting data as human readable text.
+JSON is extremely widely used, and parsers which can read and write it exist for essentially every commonly-used programming language.
+
+JSON Schema, distinctly, is built *on top* of JSON, in that JSON [schemas](#schema) are themselves JSON objects which describe other JSON objects.
```
Comment (jdesrosiers) [inline learn/glossary.md:44]: Schemas can describe any JSON type, not just objects.

```suggestion
JSON Schema, distinctly, is built *on top* of JSON, in that JSON [schemas](#schema) are themselves JSON objects which describe other JSON values.
```

**Comment by Julian** [comment]
Thanks, applied both suggestions I think!

**Review by jdesrosiers — APPROVED** [review]

**Pull request merged**